src/server/script_mgr.cc

```diff
@@ -202,6 +202,10 @@ io::Result<string, GenericError> ScriptMgr::Insert(string_view body, Interpreter
   if (!params)
     return params.get_unexpected();
 
+  auto async_body = Interpreter::DetectPossibleAsyncCalls(body);
```

Comment:
I think it's better we keep both the original and the processed versions. 
we have "script list" that can help us to display both and then we will be able to debug possible bugs in prod